yt/python/yt/cli/strawberry_parser.py

```diff
@@ -17,7 +28,7 @@ def handler(**kwargs):
         cluster_proxy = yt.config["proxy"]["url"]
         params = kwargs
 
-        if cluster_proxy not in proxy_choices:
+        if _normalize_proxy(cluster_proxy) not in map(_normalize_proxy, proxy_choices):
```

Comment:
Normalization in this check is not sufficient. You will pass through a `bad cluster proxy choice` error, but you will get another less clear error (404 not found or something like this), because it is used to form a url to send queries (`https://{controller_address}/{proxy}`). So, if you want to normalize `cluster_proxy`, you should actually change it and pass on to strawberry.make_request_generator. And that's also the reason why it's wrong to normalize `proxy_choices` here (you can normalize them manually in strawberry controller config, though)

Reply:
Yeah, it's harder than I expected, so I'll wait for the issue to be resolved then.